src/utils/styles.js

```diff
@@ -7,6 +7,10 @@ const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(d
 
 export default {
 
+  merge() {
+    return ImmutabilityHelper.merge.apply(this, arguments);
```

Comment:
Shouldn't we update https://github.com/callemall/material-ui/blob/master/src/mixins/style-propable.js#L19?